app.js

```diff
@@ -77,12 +76,13 @@ router.post('/account', async ctx => {
     // TODO: this is using alice account to create all accounts. We may want to change that.
     const body = ctx.request.body;
     const newAccountId = body.newAccountId;
-    const newAccountPublicKey = body.newAccountPublicKey;
+    // TODO: we should find a way to store the key pair for the new account
```

Comment:
I don't get this. This method previously allowed us to create and store key pair client side and just send a public key that should have access to new account. Why do we have to use default key now?

Reply:
Because I don't see a way to store the keypair in the keystore. The keypair will be used to sign transaction from that newly created account, which is what `deployContract` depends on.

Reply:
Unless we want to pass the private key here, but I feel like there is some security issue with that

Reply:
there is no need to store it in server keystore. This is used by NEARStudio, it stores key client-side.

Reply:
But the signer needs to look up the private key to sign transactions

Reply:
I think I now understand how this things works. Reverted my change

Reply:
We will need to have nearstudio use nearlib directly to deploy instead of calling contract helper, since contract helper doesn't have this contract-specific key.